ruby/vampires.rb

```diff
@@ -0,0 +1,55 @@
+Vampire_Names = "dra drac ula cula fang nosf eratu"
```

Comment:
Ruby variables are never capitalized - capitalization is saved for things like class or module names.